notebooks/IFU_cube_continuum_fit/NGC4151_FeII_ContinuumFit.ipynb

```diff
@@ -224,7 +224,7 @@
    "outputs": [],
```

Comment:
<p>Thanks, @rosteen, for your work! The selecting subsets in this notebook relies on user interaction with the UI. I'm wondering if you could add to check if subset already exists and continue as is, but if it doesn't then incorporate API calls to handle this.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jdat_notebooks/pull/192/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>